packages/common/src/Adaptor.js

```diff
@@ -267,12 +267,32 @@ export function asData(data, state) {
  * the state's references.
  * @public
  * @function
- * @example
- * each("$.[*]",
- *   create("SObject",
- *     field("FirstName", sourceValue("$.firstName"))
+ * @example <caption>Inserting patitent data using lazy state. (Only in v2)</caption>
```

Comment:
$.data is lazy state loading and it doesn't work in v1. So i am bit worried there. I am not sure when officially we are going shutdown v1

Reply:
V1 will keep running for ages, but no-one is writing new jobs on it. Check with Aleksa.

Reply:
Paging @aleksa-krolls 🔔Can we document stuff that won't work on v1?

 I think we should be focusing our docs on V2. I also think talking about v1 and V2 is confusing for users. V1 of what?

Reply:
@josephjclark I agree that we should focus the docs on v2... so no need to say `Only in v2` as that should be assumed. 

That said, I _do_ think it's okay to document when an old example is from `from/supported in openfn v1` ... because this helps our team and v1 users understand why old job code may look different than what they're seeing on the latest docs. But I'll leave this up to you and @mtuchi to decide.

Reply:
That's a good steer, thanks. I'll double check this and see if I want to flag anything before merging, although I suspect not in this work